core/src/main/scala/org/broadinstitute/dsde/rawls/config/MultiCloudWorkspaceConfig.scala

```diff
@@ -43,6 +44,14 @@ case object MultiCloudWorkspaceConfig {
                 entry.getKey -> entry.getValue.unwrapped().asInstanceOf[String]
               }
               .toMap,
+            azc
+              .getConfig("costSavingLandingZoneParameters")
+              .entrySet()
+              .asScala
+              .map { entry =>
+                entry.getKey -> entry.getValue.unwrapped().asInstanceOf[String]
```

Comment:
should these be `String`s? It looks like in rawls.conf the values are all `Boolean`s, though in the test-only reference.conf you have some Strings in there. If you think there's a future use case where we need to pass strings down to WSM/LZ or to Azure for these config settings, leaving this as `String` is fine.

Reply:
This was a mistake, we will be doing future LZ params which might make more sense as strings but not under costSavingLandingZoneParameters. I'll fix it.

Reply:
Oh no wait I'm wrong, the expected params for all LZ params are string so we should match